setup.cfg

```diff
@@ -0,0 +1,55 @@
+[metadata]
+name = browser-history
+version = attr: browser_history.__version__
+description = A python module to extract browser history
+long_description = file: README.md
+long_description_content_type = text/markdown
+url = https://github.com/pesos/browser-history
+author = Samyak Sarnayak
+author_email = samyak201@gmail.com
+license = Apache License 2.0
+license_files = LICENSE
+classifiers =
+    Programming Language :: Python :: 3
+    License :: OSI Approved :: Apache Software License
+    Operating System :: OS Independent
+project_urls =
+    Documentation = https://browser-history.readthedocs.io/en/stable/
+    Source = https://github.com/pesos/browser-history/
+    Tracker = https://github.com/pesos/browser-history/issues
+
+[options]
+include_package_data = true
+packages = find:
+python_requires = >=3.6
+zip_safe = true
+
+[options.packages.find]
+exclude =
+    tests
+
+[options.entry_points]
+console_scripts =
+    browser-history = browser_history.cli:main
+
+[bdist_wheel]
+universal = true

```

Comment:
According to [this](https://packaging.python.org/guides/distributing-packages-using-setuptools/#wheels), `universal` must only be set when the project supports both python2 and python3. Since browser-history only works on >=3.6, this shouldn't be set.
The resulting wheel is named `browser_history-0.3.1-py2.py3-none-any.whl` with this set to true, which may lead to wrong compatibilities.

Reply:
Dang, this really flew off past me. So to be fair, I'll remove this header as it makes no sense for our current use case.